src/linker/implementation_metadata.yaml

```diff
@@ -1,12 +1,16 @@
 pvs_like_python:
   step: pvs_like_case_study
-  path: /mnt/team/simulation_science/priv/engineering/er_ecosystem/containers/
+  path: /mnt/team/simulation_science/priv/engineering/er_ecosystem/containers/pvs_like_python
+  name: image
```

Comment:
Should this say pvs_like_python?

Reply:
The name can now be anything (this is so the images are reusable for milestone 2)